.github/dependabot.yml

```diff
@@ -5,10 +5,16 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+  labels:
+    - "A:no-changelog"
+    - "V:state/compatible/no_backport"
 - package-ecosystem: gomod
   directory: "/"
   schedule:
     interval: weekly
+  labels:
+    - "A:no-changelog"
+    - "V:state/compatible/no_backport"  
```

Comment:
as of my knowledge, it is not possible to add global labels, hence, here I have to duplicate labels from the previous `package-ecosystem`

Reply:
I don't think we should be doing this for the go mod updates, they can be state breaking

Reply:
hm, I thought that the reviewer would change the label if the change is breaking, but I yeah your point makes sense, it is just better if we do not have labels for potentially breaking changes at all